codebase/data.py

```diff
@@ -0,0 +1,37 @@
+import re
+import string
+from num import Num
```

Comment:
```suggestion
from .num import Num
```
Use this format if you need to import from the same directory